pkgs/development/python-modules/zha-quirks/default.nix

```diff
@@ -20,7 +19,7 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [ aiohttp zigpy ];
-  checkInputs = [ pytestCheckHook conftest asynctest ];
+  checkInputs = [ pytestCheckHook asynctest ];
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook pytest-asyncio ];
```

Reply:
Tried it but it errored out:

```
ImportError while loading conftest '/build/source/tests/conftest.py'.
tests/conftest.py:3: in <module>
    from asynctest import CoroutineMock
E   ModuleNotFoundError: No module named 'asynctest'
```

I also tried keeping both `asynctest` and `pytest-asyncio` but that shows the original warning outputs

If we can find the fix quickly I don't mind changing this PR to fix the `checkInputs` but the main reason I made this PR is to remove `conftest` as a dependencies

Reply:
Yeah you are right. The warning is missleading.